output/influxdb/config_test.go

```diff
@@ -51,3 +52,13 @@ func TestParseURL(t *testing.T) {
 		})
 	}
 }
+
+func TestGetConsolidatedConfig(t *testing.T) {
+	t.Parallel()
+	testdata := map[string]string{
+		"K6_INFLUXDB_PROXY": "http://localhost:3128",
+	}
+	config, err := GetConsolidatedConfig(nil, testdata, "")
+	require.NoError(t, err)
+	require.Equal(t, config.Proxy, null.StringFrom("http://localhost:3128"))
```

Comment:
```suggestion
	assert.Equal(t, "http://localhost:3128", config.Proxy.String)
```
The order is wrong here, the expected value is the first. Also I think it's better to assert over a simpler type like `string` instead of the more complex `null.String`.